src/content/en/tools/chrome-devtools/profile/network-performance/resource-loading.markdown

```diff
@@ -138,133 +127,137 @@ Each requested resource is added as a row to the Network table, which contains t
-with a red vertical line.
-When the engine has completed parsing of the main document,
-the `DOMContentLoaded` event fires.
-Upon retrieving all the page's resources,
-the `load` event will fire.
+## View DOMContentLoaded and load event information
 
-![DOM event lines](imgs/dom-lines.png)
+The **Network** panel highlights two events: 
+[`DOMContentLoaded`](https://developer.mozilla.org/en-US/docs/Web/Events/DOMContentLoaded) and 
+[`load`](https://developer.mozilla.org/en-US/docs/Web/Events/load).
 
-## View network timing details for a specific resource
+`DOMContentLoaded` is fired when the initial markup of a page has been 
+parsed. It is displayed in two places on the **Network** panel:
 
-Hover your mouse over a color-coded bar in the Network waterfall view,
-or click on a resource name in the Network panel table
-to view performance information about a particular type:
+![DOMContentLoaded event on network panel](imgs/domcontentloaded.png)
 
-![Timing data](imgs/timeline-view-hover.png)
+1. The blue vertical bar in the **Overview** pane signifies the event.
+2. In the **Summary** pane you can see the exact time of the event.
 
-### HTTP headers
+`load` is fired when a page has fully loaded. It is displayed in three places:
 
-The Headers tab displays the resource's request URL, HTTP method, and response status code. Additionally, it lists the HTTP response and request headers and their values, and any query string parameters. 
+![load event on network panel](imgs/load.png)
 
-![HTTP headers](imgs/network-headers.png)
+1. The red vertical bar in the **Overview** pane signifies the event.
+2. The red vertical bar in the **Requests Table** signifies the event, too.
+3. In the **Summary** pane you can see the exact time of the event.
 
-You can view HTTP headers parsed and formatted, or in their source form by clicking the **View parsed**/**View source** toggle button, respectively, located next to each header's section. You can also view parameter values in their decoded or URL encoded forms by clicking the **View decoded**/**View URL encoded** toggle button next to each query string section.
+## View details for a single resource
 
-### Resource previews
+Click on a resource name (under the **Name** column of the **Requests Table**)
+to view more information about that resource.
 
-The Preview tab displays a preview of the resource, when available. Previews are currently displayed for image resources:
+![viewing details for a single resource](imgs/network-headers.png)
 
-![Resource image preview](imgs/network-image-preview.png)
+The tabs available change depending on what type of resource you've selected,
+but the four tabs below are most common:
 
-And JSON resources:
+* **Headers**. View HTTP headers associated to the resource.
+* **Preview**. View previews of JSON, image, and text resources.
+* **Response**. View the resource's HTTP response data (if any).
+* **Timing**. View a granular breakdown of the request lifecycle for the 
```

Comment:
"View" seems repeated a lot here. Maybe just dump it and get right into the description?